eng/Tools.props

```diff
@@ -22,7 +22,7 @@
   <Import Project="$(MSBuildThisFileDirectory)dependencies.props" />
 
   <ItemGroup>
-    <PackageReference Include="$(BuildToolsPackage)" Version="$(BuildToolsPackageVersion)" />
+    <PackageReference Include="$(BuildToolsPackage)" Version="$(BuildToolsPackageVersion)" GeneratePathProperty="true" />
```

Comment:
Just out of curiosity, what does `GeneratePathProperty` do?

Reply:
It sets a property in the NuGet generated props file so that you don't need to manually build/guess the path to the package.  Have a look here: https://github.com/dotnet/corefx/pull/35677/files#diff-bb9dcc3b69f6c37d64839c785095e9a6R8

Reply:
It generates an MSBuild property based on the pkg name, something like $(Pkg_Microsoft_Buildtools) that points to the package location and it takes into consideration the fallback folder, so that you don't have to figure out where the package was restored to.

That property is defined in the nuget autogenerated props file that gets autoimported.

Reply:
See: https://github.com/dotnet/corefx/pull/35677/files/bbe99d350711ebebc95329bf1a5ad96a3d48496e#diff-bb9dcc3b69f6c37d64839c785095e9a6R8

Reply:
Ah, when I reviewed your comment didn't show. Sorry for the spam.